app/src/main/java/fr/free/nrw/commons/upload/structure/depictions/DepictedItem.kt

```diff
@@ -0,0 +1,31 @@
+package fr.free.nrw.commons.upload.structure.depictions
+
+import fr.free.nrw.commons.wikidata.model.DepictSearchItem
+
+/**
+ * Model class for Depicted Item in Upload and Explore
```

Comment:
Thanks for the Javadocs!

Reply:
They were already there and survived the migration to kotlin